extensions/resource-deployment/package.json

```diff
@@ -338,14 +338,16 @@
     ]
   },
   "dependencies": {
+    "promisify-child-process": "^3.1.1",
```

Comment:
This can be done without needing a package `utils` has a promisify function that can wrap childprocess. the newer node version might even just have a promise process by default.

Reply:
it does not quite work as well and the default utils.promisify does not handle callbacks with 3 arguments by default.

Reply:
won't fix for now. I will look into chaning this later, possibly after we more to newer node version.